packages/widgets/tests/src/accordionpanel.spec.ts

```diff
@@ -77,6 +77,37 @@ describe('@lumino/widgets', () => {
       });
     });
 
+    describe('#accessibility()', () => {
+      it('should create a widget ID if it does not exist', () => {
+        let panel = new LogAccordionPanel();
+        let w1 = new Widget();
+        let w2 = new Widget();
+
+        // Expects a widget ID to be created.
+        expect(!w1.id);
```

Comment:
I think you cannot just call `expect`, you need to add an assertion like

`expect(w1.id).to.be.undefined;`

For test equality, you should use `expect(a).to.equal('value');`

You can look at the [chai library](https://www.chaijs.com/api/bdd/) to see what is available.

> Could you fix the test at line 123 that probably misguided you as it was missing an assertion.

Reply:
I wanted to make sure the tests pass :smile: 
Thanks for noticing.